README.txt

```diff
@@ -1 +1 @@
-This is a basic calculator application for iOS.
```

Comment:
This is the old text, we should use the new one.

Reply:
Okay, I'll start off with the new text.